development/clear-webpack-cache.js

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+
+const fs = require('node:fs/promises');
+const path = require('node:path');
+
+const WEBPACK_CACHE_DIRECTORY = path.resolve(
+  __dirname,
+  '..',
+  'node_modules',
+  '.cache',
+  'webpack',
+);
+
+/**
+ * Clear the Webpack cache.
+ *
+ * This is typically run in the `postinstall` npm/Yarn lifecycle script.
+ */
+async function main() {
+  await fs.rm(WEBPACK_CACHE_DIRECTORY, { force: true, recursive: true });
+}
+
+main().catch((error) => {
+  console.error(error);
+  process.exitCode = 1;
+});
```

Comment:
This could be simplified if it was a `.mjs` script, so we can just use top-level await:

```suggestion
/**
 * Clear the Webpack cache.
 *
 * This is typically run in the `postinstall` npm/Yarn lifecycle script.
 */
await fs.rm(WEBPACK_CACHE_DIRECTORY, { force: true, recursive: true });
```

Reply:
I've been avoiding the use of top-level await intentionally because I saw it raised as an obstacle for LavaMoat node, which we might at some point start using for all of our development scripts. Not sure that's a reason to avoid it altogether, but.... the CommonJS version seems fine 🤷